src/components/AlbumName.jsx

```diff
@@ -0,0 +1,7 @@
+
+
+export const AlbumName = () => {
```

Comment:
Here, two properties will be passed: one for the album name and the other one for the url. E.g: `export const AlbumName = ({ name, url }) => {
    return (
        < div className="album-name">
            <a
                href={url}
                target="_blank"
                rel="noopener noreferrer">{name}</a>
        </div >
    )
}`